gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -170,7 +172,7 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
         if client_options is None:
             client_options = ClientOptions.ClientOptions()
 
-        if transport is None and client_options.api_endpoint is None:
```

Comment:
Since the condition changed (now it becomes: `client_options` is used when transport is not a {{ service.name }}Transport object), please update the `client_options` document above to reflect this change.

Reply:
The docstring updated to:
```
            client_options (ClientOptions): Custom options for the client. It
                won't take effect if a ``transport`` instance is provided.
```